src/shared/ask-password-api.c

```diff
@@ -113,6 +114,26 @@ static int touch_ask_password_directory(AskPasswordFlags flags) {
         return 1; /* did something */
 }
 
+static usec_t keyring_timeout(void) {
```

Comment:
please put "cache" in the name. After all we use the keyring mostly for caching purpeses here and otherwise refer to it too under that name.

keyring_cache_timeout()